Basic/Invaders/Assets/Prefabs/shield.prefab

```diff
@@ -14,7 +14,6 @@ GameObject:
   - component: {fileID: 6100000}
   - component: {fileID: 11400000}
   - component: {fileID: 6564576217164106930}
-  - component: {fileID: 8844355272195160781}
```

Comment:
what component was removed here?

Reply:
I removed the NetworkTransform, Shields are spawned and destroyed but never move, so they don't need to sync their Transform at all.